src/cli/commands/force/lightning/local/__tests__/setup.test.ts

```diff
@@ -30,7 +30,7 @@ describe('Setup Tests', () => {
     test('Should route to Setup in lwc-dev-mobile-core', async () => {
         const setup = new Setup(
             ['-p', 'ios'],
-            new Config.Config(({} as any) as Config.Options)
+            new Config.Config({} as any as Config.Options)
```

Comment:
I didn't recognize back then when I read the code for the first time but seeing this now I feel it is bit awkward to cast to `any` and then to `Config.Options`. So question, is it possible to create an empty `Config.Options` with something like `new Config.Options()`? Then that way we can use that as the parameter instead? If there's no easy way I am fine with this.

Reply:
We cannot create a new instance of `Config.Options` b/c it is an interface not a class. But looks like, unlike TSLint, in ESLint we can just write `{} as Config.Options` and all is fine, so I'll make that change